services/src/main/java/org/keycloak/social/openshift/OpenshifV3IdentityProviderConfig.java

```diff
@@ -0,0 +1,27 @@
+package org.keycloak.social.openshift;
+
+import org.keycloak.broker.oidc.OAuth2IdentityProviderConfig;
+import org.keycloak.models.IdentityProviderModel;
+
+public class OpenshifV3IdentityProviderConfig extends OAuth2IdentityProviderConfig {
```

Comment:
How come I haven't noticed that....
![giphy 13](https://cloud.githubusercontent.com/assets/719616/24335067/b4196150-1276-11e7-806e-0dbbaffaaf3f.gif)

I will send PR in the morning :)